controllers/useraccount/useraccount_controller.go

```diff
@@ -375,6 +375,11 @@ func setLabelsAndAnnotations(object metav1.Object, userAcc *toolchainv1alpha1.Us
 		object.SetLabels(labels)
 		changed = true
 	}
+	if _, exists := labels[toolchainv1alpha1.SpaceLabelKey]; !exists {
+		labels[toolchainv1alpha1.SpaceLabelKey] = userAcc.Name
+		object.SetLabels(labels)
+		changed = true
+	}
```

Comment:
I'm confused - why do we need to set the Space label at User & Identity resources? They are not related to Spaces - one user may have access to more Spaces. I think that we can keep the owner there for now since it represents the user who owns these resources. We can discuss later if we need to rename the label also here for example to "useraccount" or something like that. Let's focus on NSTemplateSet only for now

Reply:
Ok, I agree that maybe `SpaceLabelKey` is not well suited, but I thought we wanted to get rid of `OwnerLabelKey` , so yes maybe introducing something different that refers to the user account makes more sense.

I've dropped this update for now, please see  https://github.com/codeready-toolchain/member-operator/pull/438/commits/12a90172db12e0100a5c875597112d10212f4956 and https://github.com/codeready-toolchain/member-operator/pull/438/commits/11be25099ca47a558f0ca2ef25ae98ae519b03ac